translator/translate/metrics/append_dimensions/append_dimensions_test.go

```diff
@@ -11,19 +11,19 @@ import (
 )
 
 func TestAppendDimensions(t *testing.T) {
-	e := new(appendDimensions)
+	a := new(appendDimensions)
 	//Check whether override default config
 	var input interface{}
-	err := json.Unmarshal([]byte(`{
+	e := json.Unmarshal([]byte(`{
```

Comment:
This feels like an unnecessary change. Why did you change `err` to `e`? This caused you to change the variable name of the dimension.

Reply:
It's for differentiate between error in usual function and in testing function. However, most of others companies using golang would use err in their function instead. That's why I need third party opinion a little bit.